handler/population-types.go

```diff
@@ -68,3 +116,29 @@ func (h *PopulationTypes) GetAreaTypes(w http.ResponseWriter, r *http.Request) {
 
 	h.responder.JSON(ctx, w, http.StatusOK, resp)
 }
+
+// Required for GET area types
+func (h *PopulationTypes) authenticate(r *http.Request, logData log.Data) bool {
```

Comment:
This logData is not actually being used for anything here, it should be removed

Reply:
removed